tests/io/v3/vm/server/test_server_api.py

```diff
@@ -0,0 +1,78 @@
+'''
+Testing the Server API
+'''
+
+import responses
+
+SERVER_BASE_URL = r'https://cloud.tenable.com/api/v3/server'
+BASE_URL = r'https://cloud.tenable.com'
+
+
+@responses.activate
+def test_properties(api):
+    properties_response = {
+        'analytics': {
+            'enabled': True,
+            'key': '00000000-0000-0000-0000-000000000000',
+            'site_id': 'abc',
+        },
+        'capabilities': {
+            'multi_scanner': True,
+            'multi_user': 'full',
+            'report_email_config': True,
+            'two_factor': {'smtp': False, 'twilio': True},
+        },
+        'container_db_version': '11.0.3',
+        'enterprise': True,
+        'evaluation': {'limitEnabled': True, 'targets': 25},
+        'expiration': 1769403600,
+        'expiration_time': 1500,
+        'force_ui_reload': False,
+        'idle_timeout': '30',
+        'license': {
+            'agents': 512,
+            'evaluation': False,
+            'expiration_date': 1769403600,
+            'ips': 10000,
+            'scanners': 2,
+            'users': 30,
+        },
+        'limitEnabled': False,
+        'loaded_plugin_set': '202112171226',
+        'login_banner': None,
+        'msp': True,
+        'nessus_type': 'Nessus Cloud',
+        'nessus_ui_build': '308',
+        'nessus_ui_version': '11.0.375',
+        'notifications': [],
+        'plugin_set': '202112171226',
+        'region': 'abc',
+        'scanner_boottime': 1617702260,
+        'server_build': 'D20144',
+        'server_uuid': '1b6c59f6-0000-0000-0000-01996d3b57911c81433cc5c59a7b',
```

Comment:
should probably be `id`

Reply:
@tomeara Just to confirm. This should be **server_id**. Right? Or should we only keep it as **id**?

Reply:
I think we can just keep it as `id`, since the entity is `server` as is.

Reply:
Made the changes.